matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/store/db/model/CryptoMetadataEntity.kt

```diff
@@ -38,7 +38,9 @@ internal open class CryptoMetadataEntity(
 
         var xSignMasterPrivateKey: String? = null,
         var xSignUserPrivateKey: String? = null,
-        var xSignSelfSignedPrivateKey: String? = null
+        var xSignSelfSignedPrivateKey: String? = null,
+        var keyBackupRecoveryKey: String? = null,
+        var keyBackupRecoveryKeyVersion: String? = null
```

Comment:
Now I need to add these to my schema, thank you :)

Reply:
:)